iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -365,6 +381,8 @@ public static void checkConvertersMode(Validable validable, HvdcLine.ConvertersM
     public static void checkPowerFactor(Validable validable, double powerFactor) {
         if (Double.isNaN(powerFactor)) {
             throw new ValidationException(validable, "power factor is invalid");
+        } else if (Math.abs(powerFactor) > 3) {
+            throw new ValidationException(validable, "power factor is invalid, it should be a ratio");
```

Comment:
The power factor is defined as the ratio of the active power to the apparent power
`powerFactor = P/ Math.hypot(P, Q)`
so the powerFactor is a number in the closed interval [-1,1], `-1 <= powerFactor <= 1`

Reply:
Nice thanks, I was not sure about the hupper limit. If it is 1, it is really easiest ! I am going to change the PR.